### DISCUSSION TITLE
why tmux command list-windows not autocomplete but list-sessions get ?

### DISCUSSION THREAD

**Discussion by OriLiMu** [fish-shell/fish-shell#10619] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/9cb86646-ce1c-41a7-93f7-edc8ac8b111b)
You can see the options in the list not contains "list-windows". The completion now is "list-window" because I just run the command. 
Why list-window not list below?

**Comment by zanchey** [comment, marked as answer]
Nobody has gotten around to adding it yet. In `share/completions/tmux.fish`:

```
###############  Begin: Windows and Panes ###############
#TODO - these commands are not currently implemented.
#there is a section in the tmux man page that has the same title as this section
#use the "Clients and Sessions" code as an example when implementing this
###############  End:   Windows and Panes ###############
```
